Makefile

```diff
@@ -136,8 +136,12 @@ coverage:
 	go tool cover -html=coverage.out -o coverage.html
 	open coverage.html
 
+.PHONY: woke
+woke:
+	woke -c https://via.vmw.com/its-woke-rules
```

Comment:
wdyt of using that same pattern as we're going for the other tools?

- add to [hack/tools/go.mod](https://github.com/vmware-tanzu/cartographer/blob/375fd52a60184125ada30a77d40a137199ae9a90/hack/tools/go.mod)
- `go run` it [ala golangci-lint](https://github.com/vmware-tanzu/cartographer/blob/375fd52a60184125ada30a77d40a137199ae9a90/Makefile#L1)

Reply:
Oh yeah i forgot it was written in go. Original project was javascript